README.md

```diff
@@ -12,15 +12,18 @@ npm install react-inline-confirm --save
 
 ## How to Use
 
+```IsExecuting``` is a required flag prop that can be used to track the execution state of the ```react-inline-confirm``` button's click event handler from the parent component. This can come in handy when the callback method returns a promise or performs asynchronous operations in general, since those can return before the method execution actually produces a result.
```

Comment:
single backticks when inline

[docs](https://guides.github.com/features/mastering-markdown/)

also I think we should put the explanation after the example